### PR TITLE
chore(inputs.win_wmi): Cleanup and refactor code

### PR DESCRIPTION
**Pull request by srebhan**
## Summary

Cleanup code and factor out query-code to own file.

## Checklist

- [x] No AI generated code was used in this PR

## Related issues

related to #14942

### CODE REVIEW

File: plugins/inputs/win_wmi/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+operating systems. With the win_wmi plugin, it is possible to
+capture and filter virtually any configuration or metric value exposed
+through the Windows Management Instrumentation ([WMI][WMIdoc])
+service. At minimum, the telegraf service user must have permission
+to [read][ACL] the WMI namespace that is being queried.
+
+[ACL]: https://learn.microsoft.com/en-us/windows/win32/wmisdk/access-to-wmi-namespaces
+[WMIdoc]: https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
```
Comment (powersj) [inline plugins/inputs/win_wmi/README.md:13]: Is this whole file whitespace changes?

File: plugins/inputs/win_wmi/query.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
+//go:build windows
```
Comment (powersj) [inline plugins/inputs/win_wmi/query.go:1]: Any logic changes or just copy and paste?

**Review by powersj — COMMENTED** [review]
I think I'm +1 if this is just copy/paste and whitespace changes, but my eyes are not good enough to go back and forth right now.

File: plugins/inputs/win_wmi/query.go (hunk 2 above)
Comment (srebhan) [reply plugins/inputs/win_wmi/query.go:1]: Just copy-and-paste plus dissolving some functions and copy the code to the corresponding calling location.

**Review by powersj — APPROVED** [review]

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**